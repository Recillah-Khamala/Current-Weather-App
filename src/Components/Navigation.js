const Navigation = () => <div>Navigation</div>;

export default Navigation;
