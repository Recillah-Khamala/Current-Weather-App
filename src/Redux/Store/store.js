import { ConfigureStore } from '@reduxjs/toolkit';

const Store = ConfigureStore({
  reducer: {},
});

export default Store;
