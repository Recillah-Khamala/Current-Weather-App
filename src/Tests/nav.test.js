import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { HashRouter as Router } from 'react-router-dom';
import Store from '../Redux/Store/store';
import Navigation from '../Components/Navigation';

describe('Testing by jest snapshots', () => {
  it('matches navigation  Snapshot', () => {
    const navDisplay = renderer
      .create(
        <Provider store={Store}>
          <Router>
            <Navigation />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(navDisplay).toMatchSnapshot();
  });
});