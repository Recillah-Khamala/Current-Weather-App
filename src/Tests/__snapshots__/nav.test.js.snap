// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing by jest snapshots matches navigation  Snapshot 1`] = `
<nav
  className="flex justify-between py-8 border-b bg-steel-blue text-white lg:px-12"
>
  <div
    className="w-1/3 flex lg-gap-5 items-center font-bold md-text-2xl lg-text-2xl lg-leading-10 lg-tracking-wide capitalize"
  >
    <img
      alt="logo"
      className="w-8 md-w-12 lg-w-16 h-8 md-h-12 lg-h-20"
      src="logo.png"
    />
    <div>
       Current Weather App
    </div>
  </div>
  <ul
    className="flex gap-1 lg:gap-14 items-center font-medium"
  >
    <li>
      <a
        aria-current="page"
        className="font-extrabold lg-font-black border md-border-2 lg-border-4 rounded-full bg-blue-900 p-4"
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="#/Weather"
        onClick={[Function]}
      >
        Weather
      </a>
    </li>
    <li>
      <a
        className=""
        href="#/Contacts"
        onClick={[Function]}
      >
        Contact Us
      </a>
    </li>
  </ul>
</nav>
`;
